synonymdev/bitkit-react-native
# feat(lightning): update LN onboarding flow

### Description

Updates the Lightning onboarding flow to the new version from design prototype v37.

### Linked Issues/Tasks

https://app.asana.com/0/1202659694998671/1204410439373307
https://app.asana.com/0/1202659694998671/1204361714481221

### Type of change

- New feature (non-breaking change which adds functionality)

### Tests

- Detox test

### Screenshot / Video

Channel open:

https://github.com/synonymdev/bitkit/assets/8538369/2ae4feef-3542-41a8-b7d8-3ecac2e76208

Channel close:

https://github.com/synonymdev/bitkit/assets/8538369/ee8067c3-6f80-43c6-86c7-9d89e7113c1d

### QA Notes

Buy a channel using the Quick & Custom LN onboarding flows and close them again. Check that suggestion cards are correct.
